seml/manage.py

```diff
@@ -340,3 +342,41 @@ def get_slurm_arrays_tasks():
             return {}
     except subprocess.CalledProcessError:
         return {}
+
+
+def get_nonempty_input(field_name, num_trials=3):
+    get_input = getpass if "password" in field_name else input
+    field = get_input(f"Please input the {field_name}: ")
+    trials = 0
+    while (field is None or len(field) == 0) and trials < num_trials:
+        logging.error(f'{field_name} was empty.')
+        field = get_input(f"Please input the {field_name}")
+        trials += 1
+    if field is None or len(field) == 0:
+        raise ValueError(f"Did not receive an input for {num_trials}. Aborting.")
+    return field
+
+
+def mongodb_credentials_prompt():
+    logging.info('Configuring SEML. Warning: password will be stored in plain text.')
+    host = get_nonempty_input("MongoDB host")
+    port = input('Port (default: 27017):')
+    if port == "":
+        port = "27017"
+    database = get_nonempty_input("database name")
+    username = get_nonempty_input("user name")
+    password = get_nonempty_input("password")
```

Comment:
The password will be shown in the console.

Reply:
It won't, since in this case we use `getpass` to read the password.

Reply:
Oh, I missed that. Nice!